src/components/Item/ItemCard.tsx

```diff
@@ -6,15 +6,10 @@ import Vote from '@/components/Vote/Vote';
 import { IMAGE_PATH, DEFAULT_IMAGE } from '@/constants/constantValues';
 import { MoviesShowsContext } from '@/context/Context';
 import calculateAverageVote from '@/helpers/calculateAverageVote';
-import { ItemType } from '@/types/types';
+import { ItemCardProps } from '@/types/types';
 
 import './ItemCard.css';
 
-type ItemCardProps = {
-  item: ItemType;
-  key: number;
-};
-
 export default function ItemCard({ item }: ItemCardProps) {
   const { contentType } = useContext(MoviesShowsContext);
   const { id, poster_path, title, name, vote_average } = item;
```

Comment:
You should normalize the response in the API helper in order to have all props written in camelCase. Also, please check whether this 3rd party API endpoint supports retrieving the response in camelCase notation.

Reply:
The Movie Database API doesn't support retrieving the response in camelCase notation. I will create a helper function to normalize the response.